examples/persistent-volume-provisioning/README.md

```diff
@@ -134,6 +135,8 @@ For example:
   'Distribute volume':
     `volumetype: none`
 
+* `volumeoptions` : This option allows you to specify the gluster volume option which has to be set on the provisioned volume. The value string is an abstraction of the real options set on the volume. At present the only available value which can be set by this attribute is `encryption`, if this value has been set, all the required volume options ( for eg. `client.ssl, server.ssl`) for enabling encryption on the volume will be performed by Heketi.
```

Comment:
@jsafrane has a proposal out to be able pass parameters from the PVC through the storage class to the provisioner.  Would it make more sense to have one parameter per option, instead of combining all the options into one parameter?  This could potentially make it easier to parse and validate each option, as more are added.  And it could be better for readability in general.